setup.cfg

```diff
@@ -47,7 +48,6 @@ install_requires =
     scipy
     scikit-learn
     scikit-misc
-    loompy==3.0.6
```

Comment:
Did you test if removing this is okay?

Reply:
I recreated the scib-pipeline environment from scratch and everything ran through without issue. It must have been some integration method specific dependency, as it also doesn't affect the github actions test.